obspy/taup/seismic_phase.py

```diff
@@ -146,37 +152,68 @@ def parse_name(self, tMod):
         currLeg = self.legs[0]
         nextLeg = currLeg
         isPWave = True
+        isPWavePrev = isPWave
+        endAction = self.TRANSDOWN
+
```

Comment:
In this case, `endAction` was class-level, and I moved it into this function because it's not really used anywhere else. I mostly just dropped `self` and didn't bother renaming it.